platforms/android/include.gradle

```diff
@@ -38,3 +38,5 @@ dependencies {
 //    compile "com.google.android.gms:play-services-auth:9.6.+"
 
 }
+
+apply plugin: "com.google.gms.google-services"
```

Comment:
Sadly on my machine this leads to:

```
* Where:
Script '/Users/eddyverbruggen/sandboxes/_remove/firebasepanayot/platforms/android/configurations/nativescript-plugin-firebase/include.gradle' line: 42

* What went wrong:
A problem occurred evaluating script.
> Plugin with id 'com.google.gms.google-services' not found.
```

I'm running tns 2.3.0.. perhaps this changed in a more recent version?

Still applying the plugin at the bottom of `platforms/android/build.gradle` works fine though.